dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -374,9 +374,9 @@ private String calculateReverseComplement(String alleleInReverseStrand) {
     public Region getVariantCoordinates() {
         Region variantRegion;
         if (isValidRegion(chromosomeRegion)) {
-            variantRegion = chromosomeRegion;
+            variantRegion = new Region(chromosomeRegion.getChromosome(), chromosomeRegion.getStart(), chromosomeRegion.getEnd());
```

Comment:
Why are we returning a copy object?

Reply:
Because the assigned object was getting updated every time we called this method, thus causing the +1 coordinate problem.

Reply:
alright. i am closing my request since this answers my question.

Reply:
to leave a reference, see c47ed23 and 31f9d0f